test/RibbonThetaVault.ts

```diff
@@ -139,15 +153,13 @@ describe("RibbonThetaVault", () => {
     chainlinkPricer: CHAINLINK_WETH_PRICER_NEW[chainId],
     deltaFirstOption: BigNumber.from("1000"),
     deltaSecondOption: BigNumber.from("1000"),
-    deltaStep: BigNumber.from(chainId === CHAINID.AVAX_MAINNET ? "10" : "100"),
+    deltaStep: BigNumber.from(chainId === CHAINID.ETH_MAINNET ? "100" : "10"),
```

Comment:
same

Reply:
Say as above, `Oh on Avalanche, it's really AVAX as the WETH address.`